python/lsst/ts/linearstage/controllers/zaber_lst/zaber_lst.py

```diff
@@ -123,45 +123,48 @@ def __init__(self, simulation_mode, log) -> None:
         }
 
         self.simulation_mode = bool(simulation_mode)
+        self.configure(config)
         self.log.debug(
             f"Initialized ZaberLSTStage, simulation mode is {self.simulation_mode}"
         )
 
+    @property
+    def connected(self):
+        return self.commander is not None
+
     def configure(self, config):
         """Configure the settings.
 
         Parameters
         ----------
         config : `types.SimpleNamespace`
         """
-        self.steps_conversion = config.steps_per_mm
+        self.config = types.SimpleNamespace(**config["stage"])
+        self.steps_conversion = self.config.steps_per_mm
         if not self.simulation_mode:
-            self.port = config.serial_port
-            self.address = config.daisy_chain_address
+            self.port = self.config.serial_port
+            self.address = self.config.daisy_chain_address
```

Comment:
Please don't duplicate the info. Just use self.config.x when you want config item x.